externals/kyuubi-spark-sql-engine/src/main/resources/engine-event.sql

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE IF NOT EXISTS `engine_event_summary` (
+  `cluster` varchar(50) DEFAULT NULL COMMENT 'kyuubiServe集群名称，kyuubi展示服务可以管理多个集群的kyuubiServer',
```

Comment:
Please using English in Kyuubi codebase.

Reply:
done